packages/radix-ui-themes/src/components/callout.tsx

```diff
@@ -75,7 +80,7 @@ const CalloutIcon = React.forwardRef<CalloutIconElement, CalloutIconProps>(
 CalloutIcon.displayName = 'CalloutIcon';
 
 type CalloutTextElement = React.ElementRef<'p'>;
-type CalloutTextProps = Omit<React.ComponentPropsWithoutRef<typeof Text>, 'asChild'>;
+type CalloutTextProps = ExtractPropsForTag<typeof Text, 'p'>;
```

Comment:
I'm wondering if `p` is the right tag for callout

Reply:
Maybe `<div role="note">` is a bit more correct?

Reply:
Don't think so, let's keep `p` for now.